README.md

```diff
@@ -236,3 +283,8 @@ Not yet, but this is planned.
 ### Does beehive-flow work with any types of package other than NPM?
 
 Not at this stage.
+
+### Why does the Jenkinsfile example above run `beehive-flow publish` via npx instead of yarn?
+
+When `npm publish` is run from within yarn, yarn sets an environment variable to point to the yarn repo URL, not the NPM one. 
```

Comment:
By default, `npx` will install the latest version to run the command with. Can we add something here to explicitly talk about the precautions we've taken (I'm assuming `--no-install`) to make sure that using npx won't break our beehive version locking? Alternatively, can we clear the environment variable within beehive before spawning the npm process so that the user doesn't need to think about this?

Reply:
Good point. As discussed out-of-band, I've decided to run `yarn config set registry https://registry.npmjs.org/ -g` on all of the build nodes as the way of working around this weird yarn behaviour.